examples/param_est.ipynb

```diff
@@ -0,0 +1,234 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The goal of this notebook is to instruct ProgPy users on how to use the estimate_params feature for PrognosticModels.\n",
+    "\n",
+    "First some background. Parameter estimation is used to tune a general model to match the behavior of a specific system. For example, parameters of the general battery model can be used to configure the model to describe the behavior of a specific battery.\n",
+    "\n",
+    "Generally, parameter estimation is done by tuning the parameters of the model so that simulation best matches the behavior observed in some available data. In ProgPy, this is done using the prog_models.PrognosticsModel.estimate_params() method. This method takes input and output data from one or more runs, and uses scipy.optimize.minimize function to estimate the parameters of the model. For more information, refer to our Documentation (https://nasa.github.io/progpy/prog_models_guide.html#parameter-estimation)\n",
+    "\n",
+    "A few definitions:\n",
+    "* __`keys`__ `(list[str])`: Parameter keys to optimize\n",
+    "* __`times`__ `(list[float])`: Array of times for each sample\n",
+    "* __`inputs`__ `(list[InputContainer])`: Array of input containers where inputs[x] corresponds to times[x]\n",
+    "* __`outputs`__ `(list[OutputContainer])`: Array of output containers where outputs[x] corresponds to times[x]\n",
+    "* __`method`__ `(str, optional)`: Optimization method- see scipy.optimize.minimize for options\n",
+    "* __`error_method`__ `(str, optional)`: Method to use in calculating error. See calc_error for options\n",
+    "* __`bounds`__ `(tuple or dict, optional)`: Bounds for optimization in format ((lower1, upper1), (lower2, upper2), ...) or {key1: (lower1, upper1), key2: (lower2, upper2), ...}\n",
+    "* __`options`__ `(dict, optional)`: Options passed to optimizer. See scipy.optimize.minimize for options"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will show an example demonstrating the model parameter estimation feature.\n",
+    "\n",
+    "First, we will import the necessary modules from the ProgPy Package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from prog_models.models.thrown_object import ThrownObject"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can build a model with your best guess in parameters.\n",
+    "\n",
+    "We will use a ThrownObject Model and guess that our thrower is 20 meters tall. However, given our times, inputs and outputs, we can clearly tell this is obviously not true! Let's see if parameter estimation can fix this!"
```

Comment:
This comment confused me a bit because we don't have times, inputs, or outputs at this point. Maybe we could reword it? It is an unreasonable height for a human, I think that is enough